packages/vm/src/buildBlock.ts

```diff
@@ -62,7 +62,7 @@ export class BlockBuilder {
   /**
    * Calculates and returns the transactionsTrie for the block.
    */
-  private async transactionsTrie() {
+  public async transactionsTrie() {
```

Comment:
These methods (so also `logsBloom`, `receiptTrie`) were possibly `private` to avoid that people use the intermediate trie roots. (Maybe we should create an alias `intermediateTransactionsTrie()` and similar for others to signal that these are intermediate?)

Reply:
Can't we just solve this by some additional doc comment(s)?

Reply:
Yep that's fine also, you are right we don't want to bloat the code very much